ci-operator/step-registry/telcov10n/functional/cnf-ran/hub-config/telcov10n-functional-cnf-ran-hub-config-commands.sh

```
@@ -2,131 +2,107 @@
     fi
-
-    if [ ! -d "$directory" ]; then
-        echo "Error: '$directory' is not a valid directory"
-        return 1
+    local content
+    content=$(cat "$filename")
+    local varname
+    varname=$(basename "${filename}")
+    if [[ "$content" == *$'\n'* ]]; then
+      echo "${varname}: |"
+      echo "$content" | sed 's/^/  /'
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)